sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/__init__.py

```diff
@@ -52,8 +52,11 @@
 # ai.projects, but we also don't want to force users installing ai.evaluations to pull
 # in ai.projects. So we only import it if it's available and the user has ai.projects.
 try:
-    from ._converters._ai_services import AIAgentConverter
+    from ._converters._ai_agent_converter import AIAgentConverterFactory, AIAgentConverter, LegacyAgentConverter, FDPAgentConverter
     _patch_all.append("AIAgentConverter")
+    _patch_all.append("LegacyAgentConverter")
```

Comment:
Is it possible to not expose these ? And have AIAgentConverter initialize the required converter under the hood ?